cli-commands/compiler/etherlime-compile/compilerSupplier.js

```diff
@@ -5,7 +5,7 @@ const request = require('request-promise');
 const requireFromString = require('require-from-string');
 const findCacheDir = require('find-cache-dir');
 const originalRequire = require('original-require');
-const solcWrap = require('./solcWrap.js');
+const solcWrap = require('solc/wrapper');
```

Comment:
Can you explain to me why is this now used like this, or maybe why was the other one used previously?